src/reducers/plugin.js

```diff
@@ -0,0 +1,18 @@
+import { actions } from '../constants';
+import { handleActions } from 'redux-actions';
+import { fromJS } from 'immutable';
+
+const initialState = fromJS({
+  version: null,
+  error: null,
```

Comment:
Errors are set within reducers if an action is called wrong (because apparently
it's a bad idea to throw an exception within a reducer). They're picked up in
startErrorHandling and sent to sentry. They shouldn't be set during normal
usage unless something goes wrong.